frontend/src/components/eMIB/CollapsingItemContainer.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../../css/collapsing-item.css";
+
+const styles = {
+  container: {
+    position: "relative"
+  },
+  button: {
+    width: "100%",
+    textAlign: "left"
+  },
+  envelopeIcon: {
+    marginRight: 6
+  },
+  collapsingIcon: {
+    position: "absolute",
+    top: 0,
+    right: 0,
+    margin: "11px 24px 0 0",
+    color: "#00565E",
+    pointerEvents: "none"
+  },
+  contentContainer: {
+    margin: 12
+  }
+};
+
+export const ICON_TYPE = {
+  email: "fas fa-envelope",
+  task: "fas fa-tasks"
+};
+
+class CollapsingItemContainer extends Component {
+  static propTypes = {
+    iconType: PropTypes.string,
+    title: PropTypes.string.isRequired,
+    body: PropTypes.object.isRequired
+  };
+
+  state = {
+    isHidden: true,
```

Comment:
`isHidden` implies that the component isn't visible at all. `isCollapsed` might make more sense.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/171/commits/79e294735a5873417ca72af001f9e7c2681aa779 (see in PR https://github.com/code-for-canada/project-thundercat/pull/171)